MergerService/src/run.cs

```diff
@@ -358,6 +360,16 @@ private void RunTask(MergeTask task, ITaskUtils taskUtils, string? managerCallba
                                     if (blob != null)
                                     {
                                         tiles.Add(new Tile(coord, blob));
+                                        currentTilesChunkBytes += blob.Length;
+
+                                        if (currentTilesChunkBytes >= this._targetTilesChunkBytesMax || tiles.Count >= this._batchSize)
```

Comment:
Please add a comment explaining that we want to "flush" the tiles in the middle of processing a batch because of the new concept of `memory` limit.

Reply:
done